pkg/printers/codeclimate.go

```diff
@@ -31,35 +31,35 @@ func NewCodeClimate(log logutils.Log, w io.Writer) *CodeClimate {
 }
 
 func (p *CodeClimate) Print(issues []result.Issue) error {
-	codeClimateIssues := make([]CodeClimateIssue, 0, len(issues))
+	ccIssues := make([]codeClimateIssue, 0, len(issues))
 
 	for i := range issues {
 		issue := issues[i]
 
-		codeClimateIssue := CodeClimateIssue{}
-		codeClimateIssue.Description = issue.Description()
-		codeClimateIssue.CheckName = issue.FromLinter
-		codeClimateIssue.Location.Path = issue.Pos.Filename
-		codeClimateIssue.Location.Lines.Begin = issue.Pos.Line
-		codeClimateIssue.Fingerprint = issue.Fingerprint()
-		codeClimateIssue.Severity = p.sanitizer.Sanitize(issue.Severity)
+		ccIssue := codeClimateIssue{}
+		ccIssue.Description = issue.Description()
+		ccIssue.CheckName = issue.FromLinter
+		ccIssue.Location.Path = issue.Pos.Filename
+		ccIssue.Location.Lines.Begin = issue.Pos.Line
+		ccIssue.Fingerprint = issue.Fingerprint()
+		ccIssue.Severity = p.sanitizer.Sanitize(issue.Severity)
```

Comment:
Nit: Any reason to use this instead of this?


```suggestion
		ccIssue := codeClimateIssue{
			Description: issue.Description(),
			CheckName: issue.FromLinter,
			Location.Lines.Begin: issue.Pos.Line,
			Fingerprint: issue.Fingerprint(),
			Severity: p.sanitizer.Sanitize(issue.Severity)
		}
		ccIssue.Location.Path = issue.Pos.Filename
```

(Pseudo code)